docs/modules/clients/pages/clc.adoc

```diff
@@ -0,0 +1,350 @@
+= Hazelcast Command-Line Client
+:page-api-reference: https://github.com/hazelcast/hazelcast-commandline-client
+:page-toclevels: 3
+[[clc-client]]
```

Comment:
```suggestion
:page-beta: true
```

This will add a 'beta' label to the title at the top of the page.

Reply:
Great, added it https://github.com/hazelcast/hz-docs/pull/462/commits/f8ae9313c19cc809466021a2dae1a074fcb090f2